backend/src/database/repositories/filters/queryTypes.ts

```diff
@@ -0,0 +1,28 @@
+export interface QueryInput {
+  filter: any
+  orderBy: any
+  limit?: string | number
+  offset?: string | number
+  fields?: false | string | string[]
+  include?: any
+}
+
+export interface QueryOutput {
+  ['where']?: any
+  ['having']?: any
```

Comment:
Out of curiosity: what's the difference here between using it as `where?: any` vs` ['where']: any `?

Reply:
There does not seem to be one 😄
Moving to `where?: any`